usmqe/web/tendrl/cluster_work.py

```diff
@@ -0,0 +1,230 @@
+"""
+Description: Simple cluster auxiliary methods and classes
```

Comment:
What is the difference between `cluster_work` and `clusters` here? Both have the same description, moreover class named `ClustersWorkBase` is in `clusters` and not `cluster_work`. Would it make sense to merge both modules into single one?